openshift/ansible/__init__.py

```diff
@@ -1,14 +1,19 @@
-import copy
 import json
 import re
 import string
 
 from ansible.module_utils.basic import AnsibleModule
 
-from kubernetes import config
-from kubernetes.config.config_exception import ConfigException
+# from kubernetes import config
+# from kubernetes.config.config_exception import ConfigException
 from openshift import client
 
+#: Regular expression for grabbing version
+VERSION_RX = re.compile("V\d((alpha|beta)\d)?")
```

Comment:
Assuming this is an optimization to prevent the regex from being compiled for each instantiation. I'm not sure we'll see a case where we instantiate multiple instances, but I'm definitely going to keep this approach in mind going forward.